wg-serving/charter.md

```diff
@@ -0,0 +1,83 @@
+# WG Serving Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [wg-governance].
+
+[Kubernetes Charter README]: /committee-steering/governance/README.md
+
+## Scope
+
+Discuss and enhance the support for AI/ML inference workloads in Kubernetes.
+
+
+### In scope
```

Comment:
mixing of batch/training and inference could be in scope, or do we want to leave that to wg-batch?

Reply:
I think we should describe in scope as "being able to run serving workloads safely while giving up available slack capacity to batch frameworks"?

Reply:
added